src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php

```diff
@@ -151,7 +148,7 @@ public function toDataSource($mixins = [])
             'summary' => $this->getTitle(), //Again, freaking NIPSWeb
             'title' => $this->getTitle(),
             'managedid' => $this->getID(),
-            'length' => CoreUtils::happyTime($this->getLength() > 0 ? $this->getLength() : 0, true, false),
+            'length' => $this->getLength(),
```

Comment:
Well, it comes out as a hh:mm:ss.ms so I'm not sure int will work. However I will add a substring thing here.